include/ddsrouter/participant/implementations/rtps/impl/DiscoveryServerParticipant.ipp

```diff
@@ -354,7 +354,8 @@ void DiscoveryServerParticipant<ConfigurationType>::enable_tls(
         eprosima::fastdds::rtps::TCPTransportDescriptor::TLSConfig::TLSOptions::NO_SSLV2); // not safe
 
     // CA certificate
-    descriptor->tls_config.verify_file = get_param(tls_config, TLS_CA_TAG);
+    descriptor->tls_config.verify_file = tls_configuration_->certificate_authority_file();
```

Comment:
Maybe we should check if some required TLS configuration entries are set depending on the TLS configuration kind.

Reply:
this is done in `is_valid` method